src/modules/Brightness/Module.js

```diff
@@ -20,13 +22,10 @@ module.exports = function Brightness(options,UI){
         var step = this;
 
         function changePixel(r, g, b, a){
-	  options.brightness = 
-	  options.brightness || 100
-            var val = (options.brightness)/100.0
 
-            r = val*r<255?val*r:255
-            g = val*g<255?val*g:255
-            b = val*b<255?val*b:255
+            r = 255 * Math.min(val, 1)
```

Comment:
Also, I don't get this, shouldn't this be like `r * val` ??

Reply:
Yes, Sorry that's my bad. I will correct this. Thanks!